frontends/web/src/routes/exchange/components/buysell.tsx

```diff
@@ -105,6 +105,41 @@ export const BuySell = ({
     <>
       <div className={style.innerRadioButtonsContainer}>
         {!exchangeDealsResponse && <Skeleton />}
+
+        {action === 'sell' ? (
+          <div key={'btcdirect-sell'} className={style.actionableItemContainer}>
+            ----START Testing: BTC Direct sell
+            <ActionableItem
+              key={'btcdirect-sell'}
+              onClick={() => {
+                goToExchange('btcdirect');
+              }}>
+              <ExchangeProviders
+                deals={[{
+                  fee: 4.9,
+                  payment: 'card',
```

Comment:
do they support selling to card? (I don't see it on their website)

Reply:
I added a comment that this can be removed once the backend part is done.